src/components/Signup/SignUp.jsx

```diff
@@ -0,0 +1,77 @@
+// import React from 'react'
```

Comment:
Remove this commented out code

Reply:
@shakirandagire I have take the logs out as requested.
Thanks